apps/ledger-live-desktop/src/renderer/hooks/useDeeplinking.js

```diff
@@ -87,6 +87,18 @@ export function useDeepLinkHandler() {
 
       switch (url) {
         case "accounts":
+          const { address } = query;
+          if (address && typeof address === "string") {
+            const account = accounts.find(acc => acc.freshAddress === address)
+            if (account) {
+              if (account.type === "Account") {
+                navigate(`/account/${account.id}`);
+              } else {
+                navigate(`/account/${account.parentId}/${account.id}`);
+              }
+              break;
+            }
+          }
```

Comment:
please rebase on develop which probably will be an auto merge. useDeeplinking.js moved to useDeeplinking.ts

https://github.com/LedgerHQ/ledger-live/pull/3066